app/src/main/java/org/mozilla/focus/browser/cursor/CursorView.kt

```diff
@@ -58,11 +44,9 @@ class CursorView(context: Context, attrs: AttributeSet) : View(context, attrs) {
     override fun onDraw(canvas: Canvas) {
         super.onDraw(canvas)
 
-        // We draw the cursor in the top-left corner, offset so the circle isn't clipped by its
-        // parent. The translationX/Y properties will draw it into the desired position on the
-        // screen. We do it this way to avoid allocating new RadialGradients, which are
-        // constructed in relation to the current position of the circle.
-        canvas.drawCircle(radius, radius, radius, paint)
+        val cursor = context.resources.getDrawable(R.drawable.pointer, context.theme)
```

Comment:
nit: onDraw gets called at 60FPS and getDrawable, called here, can read from disk – you should cache the result of this call and re-use it. I think `setBounds` could also be moved out of this function, provided you update the bounds `onLayout`